Source/GoBot/KeyValue+ViewDatabase.swift

```diff
@@ -36,103 +36,17 @@ extension KeyValue {
         // but failed - see https://github.com/VerseApp/ios/issues/29
         
         let db = database
-        let colRootMaybe = Expression<Int64?>("root")
-
-        let msgID = try row.get(db.colMessageID)
         
         let msgKey = try row.get(db.colKey)
         let msgAuthor = try row.get(db.colAuthor)
 
-        var c: Content
-           
-        let type = try row.get(db.colMsgType)
-        
-        switch type {
```

Comment:
Ah, nice work breaking this messy code into smaller pieces 👍 It is so much easier to read now.